[2024-10-10T02:13:14.888+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-10T02:13:14.962+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_ETL_JsonData.check_folder scheduled__2024-10-09T00:00:00+00:00 [queued]>
[2024-10-10T02:13:14.984+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_ETL_JsonData.check_folder scheduled__2024-10-09T00:00:00+00:00 [queued]>
[2024-10-10T02:13:14.985+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-10-10T02:13:15.039+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_folder> on 2024-10-09 00:00:00+00:00
[2024-10-10T02:13:15.067+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_ETL_JsonData', 'check_folder', 'scheduled__2024-10-09T00:00:00+00:00', '--job-id', '873', '--raw', '--subdir', 'DAGS_FOLDER/ETLProject/main.py', '--cfg-path', '/tmp/tmp2j0zb8s7']
[2024-10-10T02:13:15.075+0000] {standard_task_runner.py:91} INFO - Job 873: Subtask check_folder
[2024-10-10T02:13:15.079+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=189) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-10T02:13:15.087+0000] {standard_task_runner.py:63} INFO - Started process 223 to run task
[2024-10-10T02:13:15.234+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_ETL_JsonData.check_folder scheduled__2024-10-09T00:00:00+00:00 [running]> on host ecb02e117702
[2024-10-10T02:13:15.442+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag_ETL_JsonData' AIRFLOW_CTX_TASK_ID='check_folder' AIRFLOW_CTX_EXECUTION_DATE='2024-10-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-09T00:00:00+00:00'
[2024-10-10T02:13:15.447+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-10T02:13:15.560+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-10T02:13:15.561+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/ETLProject/main.py", line 191, in check_folder_new
    with open(processed_file_list, 'a') as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'dag/processed_list.txt'
[2024-10-10T02:13:15.603+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_ETL_JsonData, task_id=check_folder, run_id=scheduled__2024-10-09T00:00:00+00:00, execution_date=20241009T000000, start_date=20241010T021314, end_date=20241010T021315
[2024-10-10T02:13:15.648+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 873 for task check_folder ([Errno 2] No such file or directory: 'dag/processed_list.txt'; 223)
[2024-10-10T02:13:15.716+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-10-10T02:13:15.774+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-10T02:13:15.811+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-10T04:04:03.754+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-10T04:04:03.857+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_ETL_JsonData.check_folder scheduled__2024-10-09T00:00:00+00:00 [queued]>
[2024-10-10T04:04:03.885+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_ETL_JsonData.check_folder scheduled__2024-10-09T00:00:00+00:00 [queued]>
[2024-10-10T04:04:03.888+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-10-10T04:04:03.956+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_folder> on 2024-10-09 00:00:00+00:00
[2024-10-10T04:04:03.987+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_ETL_JsonData', 'check_folder', 'scheduled__2024-10-09T00:00:00+00:00', '--job-id', '951', '--raw', '--subdir', 'DAGS_FOLDER/ETLProject/main.py', '--cfg-path', '/tmp/tmpl0vdvxg2']
[2024-10-10T04:04:03.992+0000] {standard_task_runner.py:91} INFO - Job 951: Subtask check_folder
[2024-10-10T04:04:04.051+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=6377) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-10T04:04:04.057+0000] {standard_task_runner.py:63} INFO - Started process 6399 to run task
[2024-10-10T04:04:04.277+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_ETL_JsonData.check_folder scheduled__2024-10-09T00:00:00+00:00 [running]> on host ecb02e117702
[2024-10-10T04:04:04.702+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag_ETL_JsonData' AIRFLOW_CTX_TASK_ID='check_folder' AIRFLOW_CTX_EXECUTION_DATE='2024-10-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-09T00:00:00+00:00'
[2024-10-10T04:04:04.708+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-10T04:04:04.799+0000] {logging_mixin.py:188} INFO - Check for files from today: 20241010
[2024-10-10T04:04:04.845+0000] {logging_mixin.py:188} INFO - File to process: dags/ETLProject/og_item_impression.20240830_/og_item_impression.20240830_0840_3.json.gz
[2024-10-10T04:04:04.847+0000] {logging_mixin.py:188} INFO - File to process: dags/ETLProject/og_item_impression.20240830_/og_item_impression.20240830_0950_2.json.gz
[2024-10-10T04:04:04.848+0000] {logging_mixin.py:188} INFO - File to process: dags/ETLProject/og_item_impression.20240830_/og_item_impression.20240830_1020_3.json.gz
[2024-10-10T04:04:04.850+0000] {logging_mixin.py:188} INFO - File to process: dags/ETLProject/og_item_impression.20240830_/og_item_impression.20240830_1100_1.json.gz
[2024-10-10T04:04:04.860+0000] {logging_mixin.py:188} INFO - File to process: dags/ETLProject/og_item_impression.20240830_/og_item_impression.20240830_1130_1.json.gz
[2024-10-10T04:04:04.862+0000] {logging_mixin.py:188} INFO - File to process: dags/ETLProject/og_item_impression.20240830_/og_item_impression.20240830_1930_0.json.gz
[2024-10-10T04:04:04.863+0000] {logging_mixin.py:188} INFO - File to process: dags/ETLProject/og_item_impression.20240830_/og_item_impression.20240830_2010_2.json.gz
[2024-10-10T04:04:04.867+0000] {logging_mixin.py:188} INFO - File to process: dags/ETLProject/og_item_impression.20240830_/og_item_impression.20240830_2030_3.json.gz
[2024-10-10T04:04:05.077+0000] {logging_mixin.py:188} INFO - File in folder '20240830': ['og_item_impression.20240830_0840_3.json.gz', 'og_item_impression.20240830_0950_2.json.gz', 'og_item_impression.20240830_1020_3.json.gz', 'og_item_impression.20240830_1100_1.json.gz', 'og_item_impression.20240830_1130_1.json.gz', 'og_item_impression.20240830_1930_0.json.gz', 'og_item_impression.20240830_2010_2.json.gz', 'og_item_impression.20240830_2030_3.json.gz']
[2024-10-10T04:04:05.084+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-10T04:04:05.091+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-10T04:04:05.160+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_ETL_JsonData, task_id=check_folder, run_id=scheduled__2024-10-09T00:00:00+00:00, execution_date=20241009T000000, start_date=20241010T040403, end_date=20241010T040405
[2024-10-10T04:04:05.307+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-10T04:04:05.416+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-10T04:04:05.431+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
